daemon/algod/api/server/v2/generated/model/types.go

```diff
@@ -957,11 +984,14 @@ type PostTransactionsResponse struct {
 
 // SimulationResponse defines model for SimulationResponse.
 type SimulationResponse struct {
```

Comment:
some bouncing around here between "simulate" and "simulation"

Reply:
Good point, I've renamed to "simulate" in cc7be8069